pennylane/transforms/commutation_dag.py

```diff
@@ -881,7 +863,12 @@ def __init__(self, tape):
         wires_map = OrderedDict(zip(tape.wires, consecutive_wires))
 
         for operation in tape.operations:
-            operation._wires = Wires([wires_map[wire] for wire in operation.wires.tolist()])
+            if isinstance(operation, Adjoint):
+                operation.base._wires = Wires(
+                    [wires_map[wire] for wire in operation.wires.tolist()]
+                )
+            else:
+                operation._wires = Wires([wires_map[wire] for wire in operation.wires.tolist()])
```

Comment:
This looks like a change that's not scalable, since in future we'd have to check for every symbolic class here?

Reply:
Adding a property getter and setter for `_wires`. Since `_wires` is private and shouldn't be used outside the class, this isn't a very healthy way of doing things, but I guess the `_wires` getter and setter is probably better than this logic here.